x/stakeibc/keeper/msg_server_clear_balance.go

```diff
@@ -0,0 +1,62 @@
+package keeper
+
+import (
+	"context"
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	ibctransfertypes "github.com/cosmos/ibc-go/v3/modules/apps/transfer/types"
+	"github.com/spf13/cast"
+
+	"github.com/Stride-Labs/stride/x/stakeibc/types"
+)
+
+func (k msgServer) ClearBalance(goCtx context.Context, msg *types.MsgClearBalance) (*types.MsgClearBalanceResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	zone, found := k.GetHostZone(ctx, msg.ChainId)
+	if !found {
+		return nil, sdkerrors.Wrapf(types.ErrInvalidHostZone, "chainId: %s", msg.ChainId)
+	}
+	feeAccount := zone.GetFeeAccount()
+	if feeAccount == nil {
+		return nil, sdkerrors.Wrapf(types.ErrFeeAccountNotRegistered, "chainId: %s", msg.ChainId)
+	}
+
+	// should this be a param? the transfer port _should_ always be the same across zones
```

Comment:
fwiw, I _think_ the "transfer" port is defined in the default genesis on new chains, so I think it should always be the same
```
    "transfer": {
      "port_id": "transfer",
      "denom_traces": [],
      "params": {
        "send_enabled": true,
        "receive_enabled": true
      }
    },
```

Reply:
I checked the ICS-20 spec and couldn't find anything about port names, but in ibc-go it should always be transfer. probably safe for now! https://github.com/cosmos/ibc/blob/main/spec/app/ics-020-fungible-token-transfer/README.md